.travis.yml

```diff
@@ -10,13 +10,13 @@ addons:
     sources:
       - ubuntu-toolchain-r-test
     packages:
-      - g++-8
+      - g++-7
```

Comment:
In travis the newest lcov is 1.12, and it is not compatible with gcc8. So I change to gcc7.